.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+---
+name: Problem Report
+about: Create a Report to help us improve
+---
+
+<!-- Thanks for reporting a problem for this project. READ THIS FIRST:
+
+This issue template is meant to REPORT Labrador software PROBLEMS ONLY
+
+Please DO NOT OPEN AN ISSUE:
+  - If your Labrador version is not the latest from the master branch, please update your device before submitting your issue. Your problem might already be solved. 
+The latest pre-release binaries of Labrador can be downloaded from GitHub Actions.
+
+Please take a few minutes to complete the requested information below. Our ability to provide assistance is greatly hampered without it. The details requested potentially affect which options to pursue. The small amount of time you spend completing the template will also help the volunteers providing the assistance to you to reduce the time required to help you.
+
+DO NOT DELETE ANY TEXT from this template! Otherwise the issue will be auto-closed.
+-->
```

Comment:
It might be a cultural difference, but to me this section comes across as quite hostile - I read the subtext as basically "we don't care about your problem, make sure to stroke our ego if you want anything done".

I think the best thing would be to remove it entirely.

Reply:
Above text did not display because it was inside comment-out parts. I've removed it now to avoid confusion.

But you are right, it was copied from a repo maintained by mainly Dutch and German people that don't expose a "service oriented approach"

Reply:
@EspoTek Does my reply resolve your concern? If it did, please click "resolve conversation"